core/templates/services/angular-services.index.ts

```diff
@@ -293,6 +293,7 @@ import { TranslateService } from 'services/translate.service';
 import { TranslationsBackendApiService } from 'services/translations-backend-api.service';
 import { UtilsService } from 'services/utils.service';
 import { ValidatorsService } from 'services/validators.service';
+import { PromoBarBackendApiService } from 'services/promo-bar-backend-api.service';
```

Comment:
Add this in sorted order*

Reply:
The array is in sorted order but imports are not. So, I changed it's place in array only.
Done!